test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -37,6 +37,14 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-date -Date 1/1/0030 -uformat %S%T%u%U%V%w%W%x%X%y%Y%% | Should be "0000:00:002012001/01/3000:00:00300030%"
     }
 
+    It "Passing <name> to -uformat produces a descriptive error" -TestCases @(
+        @{ name = "null"; value = $null }
+        @{ name = "an empty string"; value = ""}
```

Comment:
Suggesting removing `an` and adding whitespace after the empty string (see above)